ha.Dockerfile

```diff
@@ -54,10 +54,11 @@ COPY --chown=postgres:postgres templates/ /build/promscale/templates/
 
 RUN make package
 
-FROM timescale/timescaledb-ha:pg${PG_VERSION}-ts${TIMESCALEDB_VERSION_MAJMIN}-latest
+FROM timescale/timescaledb-ha:pg14-ts${TIMESCALEDB_VERSION_MAJMIN}-latest
```

Comment:
Given that this docker file will only be used for testing, I think this is OK because this image also contains 12,13,14. Let's just add a comment

Reply:
Sure! We can't do much about it anyway. It's a cloud's decision to have a universal image that contains all currently supported versions.

Reply:
I guess we could take their `Dockerfile` and build with our own set of ARGs but I'm not sure we really wanna do this.